tests/exo_planet_c_api.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2024 The pybind Community.
```

Comment:
I know it shows up a few other places, but I think this is frowned upon by lawyers from discussing this in context of other not-legally defined entities. What does the "The pybind Community" mean?